Arrangement_on_surface_2/include/CGAL/Arr_dcel_base.h

```diff
@@ -34,6 +34,9 @@
 #include <CGAL/Arrangement_2/Arrangement_2_iterators.h>
 #include <CGAL/assertions.h>
 
+#ifndef CGAL_ARRANGEMENT_DISABLE_UNION_SET_CCB
```

Comment:
Do you really want to keep that macro `CGAL_ARRANGEMENT_DISABLE_UNION_SET_CCB` in CGAL?

If you do, you should add a test that actually defines it.